Doc/Makefile

```diff
@@ -5,6 +5,7 @@
 
 # You can set these variables from the command line.
 PYTHON       = python3
+VENVDIR      = env
```

Comment:
I've just seen more use of `env` instead than `venv` out in the wild is all. I can put it back.